### DISCUSSION TITLE
Containerization?

### DISCUSSION THREAD

**Discussion by JoshuaFurman** [troglobit/mcjoin#27] [Q&A] [answered]
I'm curious if this tool has been tested in a container? My use case is verifying multicast-broadcasting from a Kubernetes cluster.

**Comment by JoshuaFurman** [comment, marked as answer]
My apologies I should have read the release notes prior to opening this discussion... I see the image can be found on docker hub. Thank you for this tool!

**Reply by troglobit** [reply]
There's also a package at ghcr, linked from the project's front page on GitHub.

https://github.com/troglobit/mcjoin/pkgs/container/mcjoin

Good Luck!

**Reply by JoshuaFurman** [reply]
Thank you!
